editor.planx.uk/src/@planx/components/SetFee/Editor.tsx

```diff
@@ -27,12 +43,141 @@ function SetFeeComponent(props: Props) {
+      </ModalSection>
+      <ModalSection>
+        <ModalSectionContent title="Plan✕ service charge">
+          <InputRow>
+            <Switch
+              checked={formik.values.applyServiceCharge}
+              onChange={() =>
+                formik.setFieldValue(
+                  "applyServiceCharge",
+                  !formik.values.applyServiceCharge,
+                )
+              }
+              label="Apply Plan✕ service charge"
+            />
+          </InputRow>
+          <Typography variant="body2" mb={2}>
+            A £{formik.values.serviceChargeAmount} service charge plus 20% VAT
+            will be added unless <strong>application.fee.payable</strong> is
+            less than £100 (after any Fast Track fees, exemptions, reductions or
+            increases, inclusive of VAT).
+          </Typography>
+          <Typography variant="body2" mb={2}>
+            Open Systems Lab invoices quarterly to collect the service charge.
+            If you'd like to charge a higher fee than £
+            {formik.values.serviceChargeAmount}, please contact{" "}
+            <a href="mailto:finance@opensystemslab.io" target="_self">
+              finance@opensystemslab.io
+            </a>
+            .
+          </Typography>
+        </ModalSectionContent>
+      </ModalSection>
+      <ModalSection>
+        <ModalSectionContent title="Payment processing fee">
+          <InputRow>
+            <Switch
+              checked={formik.values.applyPaymentProcessingFee}
+              onChange={() =>
+                formik.setFieldValue(
+                  "applyPaymentProcessingFee",
+                  !formik.values.applyPaymentProcessingFee,
+                )
+              }
+              label="Put payment processing fee to applicants"
+            />
+          </InputRow>
+          <Typography variant="body2" mb={2}>
+            If your council does not wish to absorb Stripe transaction fees
+            incurred by GOV.UK Pay for this service, use this option to apply an
+            additional 1% of <strong>application.fee.payable</strong> plus 20%
+            VAT payment processing fee to the amount owed by the applicant.
+          </Typography>
+          <Typography variant="body2" mb={2}>
+            Please note that it is your responsibility to configure which credit
+            card types are accepted in your GOV.UK Pay account. 1% is an average
+            processing fee only; American Express and non-EU credit cards are
+            likely to have higher rates.
+          </Typography>
+        </ModalSectionContent>
+      </ModalSection>
+      <ModalSection>
+        <ModalSectionContent title="Example" Icon={Code}>
+          <Typography variant="body2" mb={2}>
+            This example output is based on an incoming{" "}
+            <strong>application.fee.calculated</strong> of £200 and your
+            selections above.
+          </Typography>
+          <FormattedResponse
```

Comment:
I honestly think this is brilliant - I'd love more components to have an `Example` section now!